routes/api.js

```diff
@@ -1,12 +1,60 @@
 const express = require( "express" );
 const router = express.Router();
 
-// /api routes
-router.get( "/", ( req, res ) => {
```

Comment:
Ist es richtig, dass die Root-Route raus ist?

Reply:
An sich ist das beabsichtigt.

route existert nicht -> 404 via notFound error handler: https://github.com/jneidel/htw-webdev-todo/blob/85ec06d1a391fad9e51142e250d6a50b15ea23e6/app.js#L61-L62

Nur kommt aktuell ein 500, weil es vom api error handler und nicht dem not found handler bearbeitet wird.